crates/solidity/testing/snapshots/cst_output/FixedBytesType/incorrect_length/generated/0.4.11.yml

```diff
@@ -5,12 +5,12 @@ Source: >
 
 Errors: # 1 total
   - >
-    Error: Expected end of input.
+    Error: Expected FixedBytesType
```

Comment:
Wouldn't the old error be more correct? The new error indicates that it parsed `bytes32` correctly, and is expecting a new `FixedBytesType` at position 8

Reply:
If it is parsed as an identifier (as it should), I suggest reporting the error location at the start of the token, not after it.

Reply:
The issue of position of report is nuanced. Currently is shows the position at which it failed.

Reply:
Can be done in beta, but I see two issues here:

1. Error location reported on the end of the token, rather than its entire range (start/end).
2. Regression in the "actual" label, as it doesn't say that it found an `Identifier` instead.

Reply:
It doesn't even look for an Identifier. You asked it to scan a `FixedBytesType`, so that's all it checks for. Range is doable, but not in this PR.

Reply:
Sounds good. Created #335 for later.